app/jobs/prepare_person_escort_record_notifications_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class PreparePersonEscortRecordNotificationsJob < ApplicationJob
+  queue_as :notifications
+
+  def perform(topic_id:, action_name:)
+    move = PersonEscortRecord.find(topic_id).profile.move
```

Comment:
profiles can exist without moves; is there any danger here of the move being null?